core/drivers/scmi-msg/entry.c

```diff
@@ -16,6 +18,18 @@
 #include "reset_domain.h"
 #include "voltage_domain.h"
 
+/* Provision input message payload buffers for each supported entry channel */
+#define SCMI_PAYLOAD_U32_MAX	(SCMI_SEC_PAYLOAD_SIZE / sizeof(uint32_t))
+
+static uint32_t threaded_payload[CFG_NUM_THREADS][SCMI_PAYLOAD_U32_MAX]
+__maybe_unused;
+
+static uint32_t interrupt_payload[CFG_NUM_THREADS][SCMI_PAYLOAD_U32_MAX]
```

Comment:
s/CFG_NUM_THREADS/CFG_TEE_CORE_NB_CORE/ ?
Same below for fastcall_payload.

Reply:
indeed, sorry, i updates the series too quickly. I will fix.